.travis.yml

```diff
@@ -42,7 +42,7 @@ matrix:
         - PYTHON_VERSION=3.5
         - CONDA_DEPENDENCIES="setuptools sphinx cython numpy pytest-cov clang llvm-openmp"
       before_install:
-        - export TEST_OPENMP="True"
+        - export OPENMP_EXPECTED="True"
```

Comment:
I still think we don't need any of this, setting them in the env part should be good enough, just as you do in the appveyor config